hw/ip/aes/doc/checklist.md

```diff
@@ -41,7 +41,7 @@ Documentation | [DOC_INTERFACE][]       | Not Started |
 Documentation | [MISSING_FUNC][]        | Not Started |
 Documentation | [FEATURE_FROZEN][]      | Not Started |
 RTL           | [FEATURE_COMPLETE][]    | Not Started |
-RTL           | [AREA_SANITY_CHECK][]   | Not Started |
+RTL           | [AREA_CROSS_CHECK][]    | Not Started |
```

Comment:
Thanks @msfschaffner for proposing this change. @eunchan @tjaychen let me know if you prefer something different.